platform.go

```diff
@@ -177,9 +177,22 @@ func NewBindingsFromPath(path string) (Bindings, error) {
 }
 
 type vcapServicesBinding struct {
-	Name        string            `json:"name"`
-	Label       string            `json:"label"`
-	Credentials map[string]string `json:"credentials"`
+	Name        string                 `json:"name"`
+	Label       string                 `json:"label"`
+	Credentials map[string]interface{} `json:"credentials"`
+}
+
+func toJSONString(input interface{}) (string, error) {
+	switch in := input.(type) {
+	case string:
```

Comment:
Is this case needed? For `string` `json.Marshal` should do fine, shouldn't it? It will handle the `int` case as well.